src/SelectInput.js

```diff
@@ -152,16 +152,13 @@ class SelectInput extends PureComponent {
 	}
 }
 
-export default class SelectInputWithStdin extends PureComponent {
-	render() {
-		return (
-			<StdinContext.Consumer>
-				{({stdin, setRawMode}) => (
-					<SelectInput {...this.props} stdin={stdin} setRawMode={setRawMode}/>
-				)}
-			</StdinContext.Consumer>
-		);
-	}
+const SelectInputWithStdin = props => {
```

Comment:
You can remove this component and use `useStdin` hook from inside `<SelectInput>` itself.

Reply:
Addressed in bc8a16a. I've never used a hook in a class before 😲. Can you get away with this because it's a PureComponent?